### ISSUE TITLE
Train_model.py - Dataset Loading & Split

### ISSUE THREAD

**Issue by KamoMolamu** [TaalTech-Innovators/MediCareAi#18]
- Import train_test_split, config
- Load training CSV
- Fit label encoder
- Split dataset into training and validation sets
